src/_data/LiveData.js

```diff
@@ -68,6 +69,7 @@ const initAuthorized = (authData, store) => {
     api.getPayoutCurrencies();
     api.subscribeToBalance();           // some call might fail due to backend overload
     api.subscribeToAllOpenContracts();
+    api.getPaymentAgentsForCountry('id');
```

Comment:
Are they translating the country on Account Settings call? If not we can reverse from there.
If they do, need to convert it to country code instead.